src/Tasks/ContentReviewEmails.php

```diff
@@ -15,18 +15,25 @@
 use SilverStripe\SiteConfig\SiteConfig;
 use SilverStripe\View\ArrayData;
 use SilverStripe\View\SSViewer;
-use SilverStripe\ContentReview\Models\ContentReviewLog;
 
 /**
  * Daily task to send emails to the owners of content items when the review date rolls around.
  */
 class ContentReviewEmails extends BuildTask
 {
+    private static $send_emails = true;
+
     /**
      * @param HTTPRequest $request
      */
     public function run($request)
     {
+        // If the $send_emails static has been set to false, don't run the task
+        if (!$this->config()->get('send_emails')) {
+            echo 'The Content review task has been disabled';
```

Comment:
This message is not accurate, the task hasn't been disabled, the review email sending has been. I'm not sure we should output a message here in this case, might be better to log an info message. Do you know how other tasks handle output of this type?

Reply:
The sending of the email is all this task does, right? So disabling the sending of content review emails is effectively the same as disabling this task.

It seems to me that what we really want is a way to disable any `BuildTask` - which would be an enhancement to framework.

Reply:
Yep, we can look at doing a config enhancement to disable individual tasks, I think there was something happening already elsewhere, but that shouldn't block the PR here.

Reply:
That did actually get merged: https://github.com/silverstripe/silverstripe-framework/pull/10801
I'd argue that if the intention is just to not send email, and if this task effectively does nothing other than send email, then the merged feature to disable tasks makes this PR redundant.

That said if I've missed something here I'm happy for this PR to continue forward.

Reply:
Right. That will only become available in 5.1, not available for CMS 4. Or can we back-port that? Even then it's probably another 4.x minor so not within easy reach compared to having this immediately available here. Or are we constraint here not doing a module minor while waiting for CMS minor to keep in sync?

I would personally prefer to get this in and then look at updating tasks across kitchen sink to that mechanism if there were more of them that could get refactored from custom mechanism to this new approach.

Reply:
Just opened https://github.com/silverstripe/silverstripe-contentreview/pull/201 which can ship in patch here and then we can close this PR completely and wait for the new task disablement mechanism in 5.1. Backporting to 4.x question probably still valid there.

Reply:
No new features can be in CMS 4 branches for supported modules, since we're not releasing any new minors for CMS 4

Reply:
Yup